sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -328,23 +392,20 @@ These code samples show common scenario operations with the Azure Form Recognize
 The async versions of the samples (the python sample files appended with `_async`) show asynchronous operations
 with Form Recognizer and require Python 3.5 or later.
 
-Authenticate the client with a Form Recognizer API key:
-* [sample_authentication.py][sample_authentication] ([async version][sample_authentication_async])
-
-* Extract receipts: TODO
-* Extract form layouts: TODO
-* Train a model: TODO
-* Train a model with labels: TODO
-* Analyze a form using a custom model: TODO
-* Analyze a form using a custom labeled model: TODO
-* Get, list, delete models: TODO
+* Recognize receipts: [sample_recognize_receipts.py][sample_recognize_receipts] ([async version][sample_recognize_receipts_async])
```

Comment:
You're aware that these are broken links?

Reply:
yeah, all of the samples links are broken rn because I haven't merged my samples yet. Once samples are merged, I'll make sure all of the links are correctly pointing to the samples